test/e2e/specs/all-features-confirmation/record/presentation.spec.ts

```diff
@@ -662,13 +618,15 @@ test.describe('View existing record', () => {
       if (!process.env.CI) {
         await test.step('delete files', async () => {
           // delete files that may have been downloaded during tests
-          await deleteDownloadedFiles(testParams.file_names);
+          await deleteDownloadedFiles(testParams.file_names.map((name: string) => {
+            return `${process.env.PWD}/test/e2e/.download/${name}`
```

Comment:
Please use the `DOWNLOAD_FOLDER` constant (`import { DOWNLOAD_FOLDER } from '@isrd-isi-edu/chaise/test/e2e/utils/constants'`)

Reply:
Sorry, I didn't even think about that as a constant when you clearly wrote it as a "variable" in the last review